3rdparty/jvm/com/github/scopt/BUILD

```diff
@@ -0,0 +1,11 @@
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# DOCSTART: This target is included in documentation.
+jar_library(
+  jars=[
+    jar(org='com.github.scopt', name='scopt_2.11', rev='3.7.0'),
```

Comment:
This should use the `scala_jar` helper, which will append the major/minor version automatically:
```
scala_jar(org='com.github.scopt', name='scopt', rev='3.7.0')
```